static/js/modules/helpers.js

```diff
@@ -24,8 +25,22 @@ function datetime(value) {
   var parsed = moment(value, 'YYYY-MM-DDTHH:mm:ss');
   return parsed.isValid() ? parsed.format('MM-DD-YYYY') : null;
 }
+
+function prettyDatetime(value) {
```

Comment:
This is a copy and paste of the `datetime` helper, aside from the format string. How about modifying the existing function to take a `format` or `pretty` option:
```
{{{datetime this.date pretty=true}}}
```